src/site/content/en/blog/floc/index.md

```diff
@@ -168,7 +164,7 @@ cluster of thousands of similar browsing histories. These groupings aren't based
 any actual browsing histories; they're simply based on picking random centers in "cohort space" or
 cutting up the space with random lines.
 1. Each segment is given a cohort number.
-1. The web browser gets this data describing "cohort space" from its FLoC service.
+1. The web browser gets this data describing the "cohort space" from its FLoC service.
```

Comment:
Not sure about this. I meant "cohort space" as a conceptual thing like "multi-dimensional space" or whatever.

Reply:
Oh, I see. Ignore then.